Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM python:3.9
 RUN apt-get update -y
 RUN apt-get install wget build-essential python3-dev libagg-dev libpotrace-dev pkg-config libgl1 -y
 
-WORKDIR /app
-COPY requirements.txt /app
+WORKDIR /
+COPY requirements.txt /
 RUN pip install -r requirements.txt
-COPY . /app
+COPY . /
+CMD gunicorn -w 4 'vectorizing:create_app()' --timeout 0 -b 0.0.0.0:$PORT --log-level debug
```

Comment:
do we always need `debug` here?

Reply:
> do we always need `debug` here?

It's now removed from the default command, but added to the `docker-compose.yml`